webpack.config.js

```diff
@@ -3,7 +3,14 @@
 var webpack = require("webpack");
 var path = require("path");
 
+// Set up shared config to reuse in other configurations.
+var _CONFIG = {
```

Comment:
Can you pull this into it's own file and require it in.  That would make things tremendously easier for me on the generator side of things.